### ISSUE TITLE
Add timestamps for when an auction is sent to solvers and when are solutions received.

### ISSUE THREAD

**Issue by harisang** [cowprotocol/services#1596]
Since in the long run we would like to have one batch auction per block, and in order to evaluate whether we could start reducing the time limit for batch auctions in order to move towards this goal (where the time limit is currently set to, I believe 13 sec), it would be useful to get some insights about the time spent for solving of the different solvers participating in the competition.

For that, I would propose that we store this information to the database. Concretely, we could store:
1. Timestamp of when an auction is sent out to solvers.
2. Timestamps of when a valid solution was received (where a valid solution is a non-empty solution with positive score that successfully simulated; these are the solutions appearing in the solver competition table).

Currently, the only way to evaluate this is by looking at logs, but this is not really convenient, and moreover, we periodically delete logs (and also miss logs from time to time due to malfunctioning).

**Comment by vkgnosis** [comment]
We have a prometheus metric for this. See https://github.com/cowprotocol/services/pull/1592 . Not sure if we measure  non empty positive score though.

**Comment by harisang** [comment]
Ok will check to see how refined are those. The idea behind storing those in the database is that then one could make more informed decisions. E.g., here are some questions I would like to answer (i use 8 sec as an example threshold here):

1. How often do we have an auction where the last valid solution is submitted after 8 sec have passed.
2. How often is the winning solution submitted after 8 sec have passed.
3. What percent of solutions we would reject if we have a cutoff of 8 sec.
4. If we have a cutoff of 8 sec and given that we would have rejected the winner, how do the alternatives look like

etc.

In other words, since we want to move towards shorter time limits, I would like to find out more about whether we can already do this without compromising a lot.

**Comment by vkgnosis** [comment]
The current metric is enough to calculate averages per solver-outcome pairs. But outcome is not fine grained enough to distinguish winning solutions from others. The way you describe it, you do need database entries.

**Comment by MartinquaXD** [comment]
I believe we have those metrics for the colocated autopilot by now.
